hack/make-rules/test-cmd-util.sh

```diff
@@ -4474,6 +4474,11 @@ run_kubectl_all_namespace_tests() {
   # Command
   kubectl get pods --all-namespaces --namespace=default
 
+  ### Check --all-namespaces option shows namespaces
+  # Command
+  output_message=$(kubectl get pods --all-namespaces "${kube_flags[@]}")
+  kube::test::if_has_string "${output_message}" "dddd"
```

Comment:
where is this dddd namespace coming from?

Reply:
@liggitt  sorry, since I am not very clear with how this test works. What I am trying to do here is a test. Try to figure out  how to test if namespace showed correctly.

Reply:
@liggitt  thanks, done